wd_helper.au3

```diff
@@ -1345,53 +1350,15 @@ Func _WD_UpdateDriver($sBrowser, $sInstallDir = Default, $bFlag64 = Default, $bF
 						; Close any instances of webdriver
 						__WD_CloseDriver($sDriverEXE)
 
-						#Region - Extract new instance of webdriver
-						; Handle COM Errors
-						Local $oErr = ObjEvent("AutoIt.Error", __WD_ErrHnd)
-						#forceref $oErr
-						$oShell = ObjCreate("Shell.Application")
-						If @error Then
-							$iErr = $_WD_ERROR_GeneralError
-						ElseIf FileGetSize($sTempFile) = 0 Then
-							$iErr = $_WD_ERROR_FileIssue
-							$iExt = 11 ; $iExt from 11 to 19 are related to _WD_UpdateDriver()
-						ElseIf IsObj($oShell.NameSpace($sTempFile)) = 0 Then
-							$iErr = $_WD_ERROR_FileIssue
-							$iExt = 12
-						ElseIf IsObj($oShell.NameSpace($sInstallDir)) = 0 Then
-							$iErr = $_WD_ERROR_FileIssue
-							$iExt = 13
-						Else
-							Local $oNameSpace_Temp = $oShell.NameSpace($sTempFile)
-							$FilesInZip = $oNameSpace_Temp.items
-							If @error Then
-								$iErr = $_WD_ERROR_GeneralError
-							Else
-								Local $oNameSpace_Install = $oShell.NameSpace($sInstallDir)
-								Local $bEXEWasFound = False
-								For $FileItem In $FilesInZip     ; Check the files in the archive separately
-									; https://docs.microsoft.com/en-us/windows/win32/shell/folderitem
-									If StringRight($FileItem.Name, 4) = ".exe" Or StringRight($FileItem.Path, 4) = ".exe" Then     ; extract only EXE files
-										; delete webdriver from disk before unpacking to avoid potential problems
-										FileDelete($sInstallDir & $sDriverEXE)
-										$bEXEWasFound = True
-										$oNameSpace_Install.CopyHere($FileItem, 20)     ; 20 = (4) Do not display a progress dialog box. + (16) Respond with "Yes to All" for any dialog box that is displayed.
-									EndIf
-								Next
-								If @error Then
-									$iErr = $_WD_ERROR_GeneralError
-								ElseIf Not $bEXEWasFound Then
-									$iErr = $_WD_ERROR_FileIssue
-									$iExt = 19 ; $iExt from 11 to 19 are related to _WD_UpdateDriver()
-								Else
-									$iErr = $_WD_ERROR_Success
-									$bResult = True
-								EndIf
-							EndIf
-						EndIf
-						#EndRegion - Extract new instance of webdriver
+						; Extract
+						__WD_UpdateExtractor($sTempFile, $sInstallDir, $sDriverEXE)
+						If Not @error Then $bResult = True
+						$iErr = @error
+						$iExt = @extended
 					EndIf
 					FileDelete($sTempFile)
+					ShellExecute($sInstallDir)
+					Exit
```

Comment:
`ShellExecute` and `Exit` are only for easier testing together with `wd_demoa.u3`

Should be deleted before merge to master